src/tests/routes/controllers/password.test.js

```diff
@@ -0,0 +1,69 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import models from '../../../db/models';
+import app from '../../../server';
+
+// configure chai to use expect
+chai.use(chaiHttp);
+const { expect } = chai;
+
+before(async () => {
+  await models.sequelize.sync({ force: true });
+});
+let resetToken;
+const userEmail = {
+  email: 'jsmith@gmail.com',
+};
+
+describe('Forgot Password', () => {
+  it('should send the user a password reset link via email', (done) => {
+    chai.request(app)
+      .post('/api/v1/forgot-password')
+      .send(userEmail)
+      .end((err, res) => {
+        resetToken = res.body.token;
+        expect(res.status).to.equal(200);
```

Comment:
we could test for the response data.

Reply:
This has been resolved, it will be marked resolved once the PR is updated with the latest commit